### ISSUE TITLE
Admin dashboard

### ISSUE THREAD

**Issue by Connor-Skudlarek** [4human-org/murphy-ecommerce-frontend#43]
This issue is not a standalone issue. It can be treated as a mini-project. Each task in the following list needs to be complete and merged into this branch before this branch will be merged into main. Each task is going to be treated as separate issues.

- [x] Create /admin/dashboard page.jsx
- [ ] Add admin analytics
- [x] Add ability to add/remove users
- [x] Add ability to add or remove items
- [x] Add ability to update items
- [ ] Add ability to monitor donations
- [ ] Add ability to query for specific transactions

When an issue is made for one of the above, add the issue #xx to the item it belongs to. Example:

`Add admin analytics #XX`

Once the above is complete, this issue can be closed.

**Comment by Connor-Skudlarek** [comment]
First working on adding/removing users.

Can get users from DB and display. Need to add add/delete features now.

**Comment by Connor-Skudlarek** [comment]
Admin dashboard can now view all users, add users, update a user, and delete user.

Want to add a better table than the cards, but the cards will work for now.
